pkg/api/v1/types.go

```diff
@@ -1025,6 +1025,46 @@ type ReplicationControllerList struct {
 	Items []ReplicationController `json:"items" description:"list of replication controllers"`
 }
 
+// DaemonControllerSpec is the specification of a daemon controller.
+type DaemonControllerSpec struct {
+	// Template is the object that describes the pod that will be created.
+	// The Daemon Controller will create this pod on every node that matches
+	// the template's node selector (or on every node if no node selector is
+	// specified).
+	Template *PodTemplateSpec `json:"template,omitempty"`
+}
+
+// DaemonControllerStatus represents the current status of a daemon
+// controller.
+type DaemonControllerStatus struct {
+	// NodesRunningDaemon is the number of nodes running the Daemon.
```

Comment:
I emailed Brian on Monday, and used the names he suggested:

type DaemonControllerStatus struct {
	// CurrentNumberScheduled is the number of nodes that are supposed to run the daemon.
	// that are (correctly) running exactly 1 copy of the daemon.
	CurrentNumberScheduled int `json:"currentNumberScheduled"`

	// NumberMisscheduled is the number of nodes that are running the Daemon, but are
	// not supposed to run the daemon.
	NumberMisscheduled int `json:"numberMisscheduled"`

	// DesiredNumberScheduled is the total number of nodes that should be running the Daemon
	// (including nodes correctly running the daemon).
	DesiredNumberScheduled int `json:"desiredNumberScheduled"`
}

Let me know if you think these are OK